### PR TITLE
License to MIT

### PR DESCRIPTION
**Pull request by pauldendulk**

### CODE REVIEW

**Comment by pauldendulk** [comment]
Deleted the LGPL LICENSE.md and replaced it with an MIT LICENSE. This needs to be updated in some more places. Perhaps it should be added to all nuspecs. If we search the full repository there will probably be more places. Also the readme and docs need to be updated.

**Comment by janusw** [comment]
> Also the readme and docs need to be updated.

I just updated the README, using a relative path instead of a link to master.

**Comment by janusw** [comment]
> If we search the full repository there will probably be more places.

I fixed three more references to LICENSE.md that I found. Simply grepping for 'LGPL' or 'LICENSE.md' gives no more hits at this point.

**Review by janusw — APPROVED** [review]
All of this is ok with me.

File: Directory.Build.props [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <Version>3.0.0</Version>
     <NeutralLanguage>en</NeutralLanguage>
     <Copyright>Copyright © Mapsui Developers 2018</Copyright>
-    <PackageLicenseUrl>https://raw.githubusercontent.com/Mapsui/Mapsui/master/LICENSE.md</PackageLicenseUrl>
```
Comment (dotMorten) [inline Directory.Build.props:7]: For standard licenses use the expression instead:     `<PackageLicenseExpression>mit</PackageLicenseExpression>`

File: Mapsui.Layers.Tiling/Layers/TileLayer.cs [hunk 2]
```diff
@@ -1,20 +1,3 @@
-// Copyright 2008 - Paul den Dulk (Geodan)
-// 
-// This file is part of SharpMap.
-// Mapsui is free software; you can redistribute it and/or modify
-// it under the terms of the GNU Lesser General Public License as published by
-// the Free Software Foundation; either version 2 of the License, or
-// (at your option) any later version.
-//
```
Comment (dotMorten) [inline Mapsui.Layers.Tiling/Layers/TileLayer.cs:8]: Even with changing source, I still think there should be license headers and origin of copied source code

File: Directory.Build.props (hunk 1 above)
Comment (janusw) [reply Directory.Build.props:7]: See 445a52a5448472d1f7489d0f0caa0f39c155c5dc

File: Mapsui.Layers.Tiling/Layers/TileLayer.cs (hunk 2 above)
Comment (pauldendulk) [reply Mapsui.Layers.Tiling/Layers/TileLayer.cs:8]: @dotMorten I updated the headers. The license header is now part .editorconfig so with code cleanup it is easy to add it to all the files. For the purpose of this review I only did it for the changed files. Do you think this solution is okay?

File: Mapsui.Layers.Tiling/Fetcher/DataFetchStrategy.cs [hunk 3]
```diff
@@ -1,19 +1,8 @@
...
-// SharpMap is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-// GNU Lesser General Public License for more details.
-
-// You should have received a copy of the GNU Lesser General Public License
-// along with SharpMap; if not, write to the Free Software
-// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA 
+// Copyright (c) The Mapsui authors.
+// The Mapsui authors licened this file under the MIT license.
```
Comment (JBou) [inline Mapsui.Layers.Tiling/Fetcher/DataFetchStrategy.cs:2]: There is a spelling mistake: licened

Comment (pauldendulk) [reply Mapsui.Layers.Tiling/Fetcher/DataFetchStrategy.cs:2]: oops!

Comment (pauldendulk) [reply Mapsui.Layers.Tiling/Fetcher/DataFetchStrategy.cs]: Pushed a fix. Thank you very much!

**Pull request merged**